src/bundle/Resources/views/content/tabs/locations.html.twig

```diff
@@ -39,3 +39,18 @@
     {% endif %}
     </tbody>
 </table>
+
+<div class="ez-locations-box">
```

Comment:
to get something visually closer to the PM definition, the markup should be something like:

```html
<form action="/whatever/uri" method="post">
<div class="ez-list-toolbar">
    <h2 class="ez-list-toolbar-label">Location content swap</h2>
</div>
<p>
    Swap the content item at this location with another
    <button type="submit" class="ez-button ez-button-ghost">Select content item</button>
    <input type="number" placeholder="Location ID">
</p>
</form>
```